lib/list_export.rb

```diff
@@ -1,5 +1,5 @@
 class ListExport
-  TYPES = [:media, :feed, :fact_check, :explainer]
+  TYPES = [:media, :feed, :fact_check, :explainer, :dashboard]
```

Comment:
Replace `:dashboard` by `:articles_dashboard` and `:tipline_dashboard`.

Reply:
Done